Lucidtech/Las/Client.cs

```diff
@@ -983,38 +1204,53 @@ public object CreateBatch(string? name = null, string? description = null)
         /// <example>
         /// <code>
         /// Client client = new Client();
-        /// var spec = new Dictionary&lt;string, object&gt;{
+        /// var specification = new Dictionary&lt;string, object&gt;{
         ///     {"language", "ASL"},
         ///     {"version", "1.0.0"},
         ///     {"definition", {...}}
         /// };
-        /// var errorConfig = new Dictionary&lt;string, string&gt;{
-        ///     {"email", "foo@bar.com}
+        /// var environmentSecrets = new List<string>{ "las:secret:<hex-uuid>" };
+        /// var env = new Dictionary<string, string>{{"FOO", "BAR"}};
+        /// var completedConfig = new Dictionary<string, object>{
+        ///     {"imageUrl", "my/docker:image"},
+        ///     {"secretId", secretId},
+        ///     {"environment", env},
+        ///     {"environmentSecrets", environmentSecrets}
         /// };
-        /// var parameters = new Dictionary&lt;string, string&gt;{
-        ///     {"name", "Name"},
-        ///     {"description", "My awesome workflow"}
+        /// var errorConfig = new Dictionary<string, object>{
+        ///     {"email", "foo@lucid.com"},
+        ///     {"manualRetry", true}
+        /// };
+        /// var parameters = new Dictionary<string, string?>{
+        ///     {"name", name},
+        ///     {"description", description}
```

Comment:
We are escaping < and > earlier. Is this not needed here? Also should there not be a whitespace between > and {?